exporter.go

```diff
@@ -385,6 +387,58 @@ func extractVal(s string) (val float64, err error) {
 	return
 }
 
+/*
+	Valid Examples
+	id=11 addr=127.0.0.1:63508 fd=8 name= age=6321 idle=6320 flags=N db=0 sub=0 psub=0 multi=-1 qbuf=0 qbuf-free=0 obl=0 oll=0 omem=0 events=r cmd=setex
+	id=14 addr=127.0.0.1:64958 fd=9 name= age=5 idle=0 flags=N db=0 sub=0 psub=0 multi=-1 qbuf=26 qbuf-free=32742 obl=0 oll=0 omem=0 events=r cmd=client
+*/
+func parseClientListString(clientInfo string) (id float64, host string, port string, fd string, name string, age string, idle string, flags string, db string, sub string, psub string, multi string, qbuf string, qbuffree string, obl string, oll string, omem string, events string, cmd string, ok bool) {
```

Comment:
This can now return a lot less results (or simple the map plus the bool for `ok`)